### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by Gataniel**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1.1
+2.2.0
```
Comment (shir) [inline .ruby-version]: @Gataniel please update to 2.2.1. 2.2.0 has serious bugs.

File: app/views/devise/sessions/new.html.slim [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-  .col-sm-6
-    = simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-      .form-inputs
-        = f.input :email, :required => false, :autofocus => true
-        = f.input :password, :required => false
-        = f.input :remember_me, :as => :boolean if devise_mapping.rememberable?
-      .form-actions
-        = f.button :submit, "Sign in", class: 'btn-primary'
-    = render "devise/shared/links"
+.row.col-sm-6
```
Comment (AlexanderUlitin) [inline app/views/devise/sessions/new.html.slim]: @Gataniel `.col-sm-6` should be inside `.row`.

File: config/initializers/simple_form.rb [hunk 3]
```diff
@@ -66,7 +72,7 @@
   config.error_notification_tag = :div
 
   # CSS class to add for error notification helper.
-  config.error_notification_class = 'alert alert-error'
+  config.error_notification_class = 'error_notification'
```
Comment (AlexanderUlitin) [inline config/initializers/simple_form.rb]: @Gataniel why you use another class for errors?

File: spec/features/organizations_list_spec.rb [hunk 4]
```diff
@@ -9,19 +9,17 @@
     visit organizations_path
   end
 
-  subject { page }
```
Comment (AlexanderUlitin) [inline spec/features/organizations_list_spec.rb]: @Gataniel why you did this changes? I don't see any difference here.

File: config/initializers/simple_form.rb (hunk 3 above)
Comment (Gataniel) [inline config/initializers/simple_form.rb]: It is updated Simple Form. Should I leave it as default or change to old alert alert-error

Comment (AlexanderUlitin) [inline config/initializers/simple_form.rb]: @Gataniel it depends on styles that we use in forms. As I know we use bootstrap and classes `alert alert-error` will show red colored alerts. Please check it and use styles that will works from Twitter Bootstrap.

Comment (AlexanderUlitin) [inline config/initializers/simple_form.rb]: @Gataniel it depends on styles that we use in forms. As I know we use bootstrap and classes `alert alert-error` will show red colored alerts. Please check it and use styles that will works from Twitter Bootstrap.

Comment (AlexanderUlitin) [inline config/initializers/simple_form.rb]: @Gataniel it depends on styles that we use in forms. As I know we use bootstrap and classes `alert alert-error` will show red colored alerts. Please check it and use styles that will works from Twitter Bootstrap.

Comment (Gataniel) [inline config/initializers/simple_form.rb]: @AlexanderUlitin  Actually I don't see any difference...Everything seems to work fine

**Pull request merged**